docs/UserGuide.md

```diff
@@ -377,8 +377,9 @@ Action | Format, Examples
 **List all** | `list all`
 **List property** | `list property`
 **List appointment** | `list appointment`
-**Add new status** | `update INDEX new AMOUNT`<br><br> e.g.,`update 1 new 600000`
-**Update status** | `update INDEX u/STATUS`<br><br> e.g. `update 3 u/proceed`
+**Add new status** | `update INDEX u/new AMOUNT`<br><br> e.g.,`update 1 u/new 600000`
+**Proceed to the next status** | `update INDEX u/proceed`
+**Remove a status** | `update INDEX u/cancel`
```

Comment:
Just wondering if you separate out this 3 formats of update in the command summary, then should you separate them in the Command section above too? Its better to be consistent so if you want to separate out to 3 formats, then both sides should be separated. If not, both sides should be 1 format only.

Reply:
ok! changed it!